src/js/modules/subtest/prototypes/SurveyRunItemView.coffee

```diff
@@ -390,6 +392,34 @@ class SurveyRunItemView extends Backbone.Marionette.CompositeView
 #    #    @trigger "ready"
 #    @trigger "subRendered"
 
+  onShow: ->
```

Comment:
From @chrisekelley : This was added to fix bug #208 and might need to be implemented in other prototypes.